lorien/Misc/I18nParser.gd

```diff
@@ -25,7 +32,11 @@ func load_files() -> void:
 				else:
 					printerr("Key not found (make sure to use spaces; not tabs): %s" % line)
 			TranslationServer.add_translation(translation)
-		
+			locales.append(translation.locale)
+			language_names.append(name)
+	Settings.locales = locales
+	Settings.language_names = language_names
```

Comment:
Don't set them here directly in the singleton. It's better to make an inner class like this in `I18nParser`:
````
class ParseResult:
     var locales: PoolStringArray
     var language_names: PoolStringArray
````

and return that.